stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/App.kt

```diff
@@ -18,6 +18,7 @@ class App : Application() {
         super.onCreate()
         instance = this
         DebugMetricsHelper.init()
```

Comment:
Init `ChatClient` and `ChatDomain` here. All the static data needs to initialized in the `Application` class, otherwise after process death it will be lost. (E.g. bring app to the background, kill app process, locate the app in task manager).